docs/en/api/POSIX-API.md

```diff
@@ -73,181 +73,363 @@ For example, running the following commands from the `${ALLUXIO_HOME}` directory
 ```config
-alluxio.fuse.mount.alluxio.path=<alluxio_path>
-alluxio.fuse.mount.point=<mount_point>
-alluxio.fuse.mount.options=<list of mount options separated by comma>
+alluxio.user.metadata.cache.enabled=true
+alluxio.user.metadata.cache.expiration.time=2h
+alluxio.user.metadata.cache.max.size=2000000
+```
+
+The metadata is cached in the AlluxioFuse Java process heap so make sure `cache.max.size * 2KB * 2 < AlluxioFuse process maximum memory allocated`.
+For example, if AlluxioFuse is launched with `-Xmx=16GB` and metadata cache can use up to 8GB memory, then `cache.max.size` should be smaller than 4 million.
+The default max cache size is `100000` which takes about `200MB` heap space.
+
+When the data is updated via other Alluxio clients, the metadata cache needs to be updated otherwise it will be stale until reaches `alluxio.user.metadata.cache.expiration.time` (default is `10min`).
+Metadata cache can be invalidated manually through Alluxio FUSE shell which is enabled by setting `alluxio.fuse.special.command.enabled=true` in `${ALLUXIO_HOME}/conf/alluxio-site.properties`
+before mounting FUSE.
+
+Run the Fuse shell command to clear all the cached metadata：
+```console
+$ ls -l /mnt/alluxio-fuse/.alluxiocli.metadatacache.dropAll
+```
+
+Run the Fuse shell to clear the metadata cache of a specific path：
+```console
+$ ls -l /mnt/alluxio-fuse/dir/dir1/.alluxiocli.metadatacache.drop
+```
+The above command will clear the metadata cache of `/mnt/alluxio-fuse/dir/dir1`, all its ancestor directories,
+and all its descendants files or directories.
+
+Run the Fuse shell to know the total number of paths that are cached locally:
+```console
+$ ls -l /mnt/alluxio-fuse/.alluxiocli.metadatacache.size
+```
+You will get metadata cache size in file size field, as in the output below:
+```
+---------- 1 root root 13 Jan  1  1970 /mnt/alluxio-fuse/.alluxiocli.metadatacache.size
 ```
 
-For example, one can mount Alluxio path `/people` to local path `/mnt/people`
-with `direct_io,entry_timeout=7200,attr_timeout=7200` mount options when starting the Alluxio worker process:
+  {% endnavtab %}
+{% endnavtabs %}
+
+#### Data Cache
+
+Data can be cached on FUSE kernel cache and/or in Alluxio FUSE process userspace cache. When the same file is accessed from multiple clients,
+file overwrite by one client may not be seen by other clients. The data cached on the FUSE side (kernel or userspace) may be stale.
+For example, the data cached on Node A might be stale if the file is deleted and overwrite concurrently by an application on Node B.
+
+{% navtabs dataCache %}
+  {% navtab Kernel Data Cache Configuration %}
+
+FUSE has the following I/O modes controlling whether data will be cached and the cache invalidation policy:
+- `direct_io` (default): disables the kernel data cache
+- `kernel_cache`: always cache data in kernel. This should only be enabled on filesystem, whether the file data is never changed externally (not through the mounted FUSE filesystem)
```

Comment:
I'm not able to understand the explanation of the kernel_cache

Reply:
Changed, please take a look to see if it's understandable